include/cling/Utils/Casting.h

```diff
@@ -0,0 +1,56 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author: Roman Zulak
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#ifndef CLING_UTILS_CASTING_H
+#define CLING_UTILS_CASTING_H
+
+namespace cling {
+  namespace utils {
+
+    ///\brief Legally cast a pointer to a function to void*.
+    ///
+    /// \param [in] The function to cast
+    ///
+    /// \return A void* of the functions address.
+    ///
+    template <class T>
+    void* FunctionToVoid(T *fPtr) {
```

Comment:
Could you rename `fPtr` to e.g. `funptr`? Else it's an unfortunate clash with ROOT's coding conventions - yes, not relevant for cling, but that change would still be much appreciated :-)